docker/Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG CARGO_BUILD_FEATURES=default
 ARG RUST_RELEASE_MODE=debug
 
 ARG AMD_BUILDER_IMAGE=rust:${RUST_VERSION}
-ARG ARM_BUILDER_IMAGE="ghcr.io/raskyld/aarch64-lemmy-linux-gnu:v0.1.0"
+ARG ARM_BUILDER_IMAGE="ghcr.io/raskyld/aarch64-lemmy-linux-gnu:v0.2.0"
```

Comment:
Would be good if you add a link to the github repo here.

Reply:
Done